packages/suite/src/utils/wallet/accountUtils.ts

```diff
@@ -179,6 +180,18 @@ export const amountToSatoshi = (amount: string, decimals: number) => {
     }
 };
 
+export const satoshiToAmount = (amount: string, decimals = 8) => {
```

Comment:
what is the reason to not use `networkAmountToSatoshi` which is exactly the same?

Reply:
Because `amountToSatoshi` multiplies the value to get _sats_ and `satoshiToAmount` divides to get _BTC (and others)_? Or am I missing something? 🤔

Reply:
removed